api/dsr/v1/Access.md

```diff
@@ -97,7 +97,7 @@ specified in the request.
 The `results` and `documents` are merged with any cached results from previous events. New documents are
 added and existing documents are updated.
 
-Once the status is set to `completed`, `cancelled` or `denied`, then no further events will be accepted.
+Once the status is set to `completed`, `in_progress` or `pending`, then no further events will be accepted.
```

Comment:
Hm I think that once the status is set to `completed` then no further events would be accepted. But for `in_progress` and `pending` statuses, the API call tile expects a callback, correct?

Reply:
@prachikhandekar what is the difference between 'in progress' and 'pending" I have only ever sent back "in progress' or "completed".

Reply:
Honestly, the backend code treats them the exact same. So they are both "valid" statuses that behave the exact same way in our backend.